src/components/Filters/Panels.tsx

```diff
@@ -84,6 +86,9 @@ const Panels: FC<PanelsProps> = ({
     return (
       <Card
         key={index}
+        role="checkbox"
+        aria-describedby={aria_describedby_label}
+        aria-checked={isSelected ? 'true' : 'false'}
```

Comment:
Could rewrite this as `aria-checked={!!isSelected}` or even `aria-checked={isSelected}` if `isSelected` is already a boolean